src/common/SurgeSynthesizerIO.cpp

```diff
@@ -200,6 +201,20 @@ void SurgeSynthesizer::loadRaw(const void* data, int size, bool preset)
    halt_engine = false;
    patch_loaded = true;
    refresh_editor = true;
+	if (patchid < 0)	//category.size() == 0)
```

Comment:
Then indentation goes shifts suddenly 3 spaces right for no reason.

Reply:
I think many of these are because the diff has hard tabs rather than spaces, also. I think.